tests/ord-service/bench/system_bundles_test.go

```diff
@@ -0,0 +1,61 @@
+package bench
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/kyma-incubator/compass/tests/pkg/clients"
+	"github.com/kyma-incubator/compass/tests/pkg/fixtures"
+	"github.com/kyma-incubator/compass/tests/pkg/request"
+	"github.com/kyma-incubator/compass/tests/pkg/tenant"
+	"github.com/stretchr/testify/require"
+	"github.com/tidwall/gjson"
+)
+
+const (
+	tenantHeader = "Tenant"
+)
+
+func BenchmarkSystemBundles(b *testing.B) {
+	//GIVEN
+	ctx := context.Background()
+	defaultTestTenant := tenant.TestTenants.GetDefaultTenantID()
+
+	appsCount := 5
+	for i := 0; i < appsCount; i++ {
+		app := fixtures.CreateApp(fmt.Sprintf("%d", i))
+		appResp, err := fixtures.RegisterApplicationFromInput(b, ctx, dexGraphQLClient, defaultTestTenant, app)
+		require.NoError(b, err)
```

Comment:
ah, sorry, I should have seen that the first time - the `defer` should be before `NoError`